examples/src/Auction/Offchain.hs

```diff
@@ -86,7 +86,7 @@ txBid submitter offerOref bid = do
       Just deadline = A.getBidDeadline datum
       seller = A.getSeller datum
       lotPlusPreviousBidPlusNft = outputValue output
-  validityInterval <- slotRangeBefore deadline
+  validityInterval <- slotRangeBefore (deadline - 1)
```

Comment:
Shouldn't this remain `slotRangeBefore deadline`? The bid validator should accept a transaction on the deadline if the node interprets slots the same as the off-chain code.